### ISSUE TITLE
.

### ISSUE THREAD

**Issue by tiagorodrigues2** [Ezzpify/FindLowDigAccounts#11]
.

**Comment by puuska** [comment]
Yeah, now lowdig is rip 😢

**Comment by sejo153** [comment]
its working in 2017 lol i even find acc with wallet 3€

**Comment by Ghost4592** [comment]
then u r lying

**Comment by peer-f** [comment]
It was working for the last 2 weeks or so. Now they have introduced a from and it goes through Steam Support. 
The program itself still works, but getting accounts is more complicated now.

**Comment by sejo153** [comment]
@Ghost4592 Im not lying, i was using it last week and everything was woring properly, i had about 20acconts that i recovered. Then hdd broke and everything with him. So im not even trying to get accounts anymore, idk about now :)

**Comment by Ghost4592** [comment]
@sejo153 @Ezzpify  what to do now? any way to get old accounts out there?

**Comment by Ezzpify** [comment]
Buy or contact Steam support to recover your old accounts. Second option won't work unless you originally owned the account.

**Comment by sejo153** [comment]
@Ezzpify Will there be any other tools for steam from you? Like mail/brute checker or some other cracking tools ?

**Comment by puuska** [comment]
guys, it works but it need some more work from you. i will attach the screenshots from my support ticket. i recovered lowdig (2007) account with it :D

**Comment by puuska** [comment]
![screenshot-help steampowered com-2017-07-15-00-28-15](https://user-images.githubusercontent.com/11046280/28233089-b0a7f61a-68f4-11e7-9e3a-64d2ec7b50b9.png)

**Comment by ch1ddy** [comment]
What did they answer? I tried the same thing and they said I don't have enough proof.

**Comment by peer-f** [comment]
![aaa](https://user-images.githubusercontent.com/26091730/28233155-2192c792-68f5-11e7-92bc-e41709ac7cea.PNG)
I've messaged them with similar information. It's pretty retarded that you should know a gamekey from literally almost a decade ago!

**Comment by puuska** [comment]
Yeah, after this they will give you access to lowdigit account, but you must spend some time and have a little knowledge about this shit

**Comment by peer-f** [comment]
So I gotta be more creative :D Was this your first try going through the support? I even had the login name of the account but i was not able to create the email cuz it was from a polish email service that is not existing anymore.

**Comment by peer-f** [comment]
You are polish or? If you want we can look into it together.

**Comment by puuska** [comment]
Yeah, it was my first lowdig support expierience, and yeah I am from poland ;)

**Comment by peer-f** [comment]
There a couple thousand accounts of the polish provider with similar login names often. Do you mind that I add you on Steam?

**Comment by puuska** [comment]
yeah, no problem. /id/puuskago, when you add me write here so I will start steam website (client on linux do not fucking work)

**Comment by peer-f** [comment]
Added

**Comment by Ghost4592** [comment]
hay @puuska i added u i cant understand that polish language

**Comment by Jose97PY** [comment]
A few days ago I recovered a 5 digit account with @gmx.de email. I think it is a question of luck who answers the support ticket. I'm not German just wrote that to sound a little more convincing with the @gmx.de email.
![fireshot capture 44 - steam support - my recent help request_ - https___help steampowered com_en_w](https://user-images.githubusercontent.com/30203216/28242906-71b022ba-6986-11e7-905e-3f3a2816d7ac.png)

**Comment by peer-f** [comment]
Well done @Jose97PY. It seems like it really depends on the support operator. I'm wondering why there are no strict rules for them. In my opinion it's almost impossible to find such old CD-Keys. I mean if you've been offline for more than 5 Years you probably moved out to your own place.

**Comment by kominax** [comment]
@Jose97PY Could you please give me an account? Ive been trying for 7 days still no luck because the support tells me you need to show us your cd key or we won't make any changes and thats all the time :( So could u please send me a account to vingcip@gmail.com please?

**Comment by Mcprozek** [comment]
If you wanna buy 12 year old lvl 7 unrestricted accounts inbox me.
Fb : www.fb.com/mcprozek

Last 80 accs left

**Comment by trance76** [comment]
So you have to write a convincing support ticket to all steam accounts you found?

**Comment by puuska** [comment]
@trance76 yes

**Comment by tamal777** [comment]
STILL WORKS.

**Comment by Mcprozek** [comment]
Pm me on fb : www.fb.com/mcprozek

**Comment by puuska** [comment]
It works with new method :^)

**Comment by Teakies** [comment]
I was trying with over two hundred (!) emails, and all of them were wrong. 
Btw have you got any script to copy & paste emails from notepad to steam website? ty

**Comment by hldsarchive** [comment]
You guys should check russian hacking sites. They found new method.

**Comment by puuska** [comment]
@hldsarchive Are you speaking about some mode of steam that you turn on with clicking on gamepad icon?

**Comment by hldsarchive** [comment]
@puuska "Big picture" method? No xd This method is good too but there is another way to restore older accounts using some API functions.

**Comment by ghost** [comment]
@hldsarchive What method? I have searched in ruski suka blyat forums and i only found 1 method (big picture) that didn't work for the acc i found (7dig)

**Comment by Mcprozek** [comment]
If anyone want to buy inbox me on my fb : www.fb.com/mcprozek

**Comment by Russlyman** [comment]
Anyone got any new methods?

**Comment by puuska** [comment]
big
picture

**Comment by Teakies** [comment]
how does it works?


________________________________

**Comment by Russlyman** [comment]
how does one big picture

**Comment by Russlyman** [comment]
pls

**Comment by puuska** [comment]
google ;)

**Comment by Mcprozek** [comment]
If anyone want to buy pm me www.fb.com/mcprozek

**Comment by skaaier** [comment]
@bxerra Same here. I have tried the big picture method multiple times with different accounts ranging from 5 digits to 6 and 7 digits. Yet when I come to the part where I have to choose between two e-mails (one of which is the real original one) , it only shows my newly created one. Someone help? Or atleast PM us for some clarification lol.

**Comment by asankos** [comment]
give me an acccount plsss itry so long time i sleepless  plss

**Comment by ghost** [comment]
Now big picture method fixed

**Comment by xDARCKx** [comment]
> Now big picture method fixed

what how?

**Comment by xtheshades** [comment]
> > Now big picture method fixed
> 
> what how?

It redirects to steam support page so rip. Any other methods?

**Comment by KingofKFCJamal** [comment]
Don't expect any updates for this program. This method has been dead in the dirt for 1-2 years now.

**Comment by puuska** [comment]
> > > Now big picture method fixed
> > 
> > 
> > what how?
> 
> It redirects to steam support page so rip. Any other methods?

holy shit, it was working month ago :/

**Comment by denotado** [comment]
is there any other method? rip bib Picture :/

**Comment by yZuhn** [comment]
Still works in 2019 i got 2 accs like 2 weeks ago

**Comment by ghost** [comment]
> Still works in 2019 i got 2 accs like 2 weeks ago

no its fixed

**Comment by yZuhn** [comment]
> > Still works in 2019 i got 2 accs like 2 weeks ago
> 
> no its fixed

lmao i got this acc 1 week ago https://steamcommunity.com/id/2339/

**Comment by Mcprozek** [comment]
how u got?

**Comment by emrecanx** [comment]
> > > Still works in 2019 i got 2 accs like 2 weeks ago
> > 
> > 
> > no its fixed
> 
> lmao i got this acc 1 week ago https://steamcommunity.com/id/2339/

how you make it????? Method PLSSSSSSSSSSSSSS

**Comment by puuska** [comment]
are you guys dumb or smth? no one will give you full method for free when they can just sell accounts for real money without risking that valve will patch it and lock all recovered accs

**Comment by yZuhn** [comment]
> are you guys dumb or smth? no one will give you full method for free when they can just sell accounts for real money without risking that valve will patch it and lock all recovered accs

bruh there aint no real methods.Just luck with steam support

**Comment by yZuhn** [comment]
> are you guys dumb or smth? no one will give you full method for free when they can just sell accounts for real money without risking that valve will patch it and lock all recovered accs

true i can get 5$ ez for selling 1 acc

**Comment by b1ack14** [comment]
It still works in 2019 I have 5 accounts like 1 month ago

**Comment by yZuhn** [comment]
guys lowdig still works but i think we wont be getting accounts of steam support anymore they added a special field where you need to type the cd key if you added it to your acc rip

**Comment by skeshisbored** [comment]
well it fixed

**Comment by skeshisbored** [comment]
![image](https://user-images.githubusercontent.com/50412444/57380782-3b1cd580-71dc-11e9-98a1-09c2de706f9b.png)

they always asking for cd keys

**Comment by yZuhn** [comment]
> ![image](https://user-images.githubusercontent.com/50412444/57380782-3b1cd580-71dc-11e9-98a1-09c2de706f9b.png)
> 
> they always asking for cd keys

Yea its really a FeelsBadMan imma try more accounts hope i get lucky

**Comment by skeshisbored** [comment]
![image](https://user-images.githubusercontent.com/50412444/57416830-5c111500-7234-11e9-9354-e42dd8be3db3.png)
i got 1 xD but it doesnt have games i think its all about your luck

**Comment by skeshisbored** [comment]
> > ![image](https://user-images.githubusercontent.com/50412444/57380782-3b1cd580-71dc-11e9-98a1-09c2de706f9b.png)
> > they always asking for cd keys
> 
> Yea its really a FeelsBadMan imma try more accounts hope i get lucky

i think its only work for account that doesnt have games because no games = no cd keys

**Comment by yZuhn** [comment]
> > > ![image](https://user-images.githubusercontent.com/50412444/57380782-3b1cd580-71dc-11e9-98a1-09c2de706f9b.png)
> > > they always asking for cd keys
> > 
> > 
> > Yea its really a FeelsBadMan imma try more accounts hope i get lucky
> 
> i think its only work for account that doesnt have games because no games = no cd keys

its very difficult to find a 0 game profile but i will continue to try and find some profiles maybe i get lucky

**Comment by yZuhn** [comment]
does anyone know any new methods or

**Comment by trance76** [comment]
does anyone have a problem when creating an account? It says  I have an temporary email idk why

**Comment by yZuhn** [comment]
> does anyone have a problem when creating an account? It says I have an temporary email idk why

why would you create a account

**Comment by trance76** [comment]
> > does anyone have a problem when creating an account? It says I have an temporary email idk why
> 
> why would you create a account

So I can get the old account, thats my method, why how you do it?

**Comment by yZuhn** [comment]
> > > does anyone have a problem when creating an account? It says I have an temporary email idk why
> > 
> > 
> > why would you create a account
> 
> So I can get the old account, thats my method, why how you do it?

does it work?

**Comment by yZuhn** [comment]
> > > does anyone have a problem when creating an account? It says I have an temporary email idk why
> > 
> > 
> > why would you create a account
> 
> So I can get the old account, thats my method, why how you do it?

and to anwer your question yea it tells me i have a disposable email too wtf

**Comment by yZuhn** [comment]
hello does this still work

**Comment by AksaKing** [comment]
> hello does this still work

yeah still

**Comment by memorymanager** [comment]
some new metod?

**Comment by Mariner1421312** [comment]
Method please? uu

**Comment by rjunior-debug** [comment]
So any new methods?

**Comment by daniil1199** [comment]
guys the method is still working, but it depends on the steam support staff that will analyze your case
![Screenshot_6](https://user-images.githubusercontent.com/94711068/152192986-f171ba6f-2c50-4cd8-b1eb-5617c22d55a6.png)
![Screenshot_5](https://user-images.githubusercontent.com/94711068/152192992-7b49b672-e9e8-4746-9488-20acd085bcd4.png)

**Comment by daniil1199** [comment]
i think that it's just my luck, so i recovered 2 accounts in one day

**Comment by memorymanager** [comment]
can u translate it in english please (the text)

**Comment by daniil1199** [comment]
Hello,

We have reset your password and sent a new one to the email address you contacted us from. If you didn't receive it, check if the message went to your spam folder or trash.

For account security, access to the exchange and the Market will be closed for a certain period after Steam Support has reset the password.

Thank you for using Steam,
Anastasia

**Comment by daniil1199** [comment]
both support's replies are the same

**Comment by kuinke** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/94711068/152207108-61d78db8-db73-431d-a215-3e2c5b4f733b.png)

**Comment by daniil1199** [comment]
and i also recieved the login and the password

**Comment by HCONH2** [comment]
cool

**Comment by memorymanager** [comment]
can you translate the text you wrote to the support please

**Comment by daniil1199** [comment]
Good evening, just remebered about my old account. I wondered if it's possible to recover it, i don't have any receipt, the original email that i used to register my account is: "you write the account's original email", also i don't remember any CD-keys, but i remember that i played Half Life and CS on this account. I would be happy if you recover my account, want to play this games again.

**Comment by HCONH2** [comment]
thanks

**Comment by ChepitaX** [comment]
How many digits is that account?

**Comment by ChepitaX** [comment]
> Good evening, just remebered about my old account. I wondered if it's possible to recover it, i don't have any receipt, the original email that i used to register my account is: "you write the account's original email", also i don't remember any CD-keys, but i remember that i played Half Life and CS on this account. I would be happy if you recover my account, want to play this games again.

How many digits is that account?

**Comment by daniil1199** [comment]
7 dig

**Comment by daniil1199** [comment]
2 days ago i recovered another one

**Comment by daniil1199** [comment]
by the way

**Comment by memorymanager** [comment]
cool i will try thanks for reply

**Comment by daniil1199** [comment]
nothing to thank for

**Comment by Karnikov235** [comment]
kuinke Can you be contacted while you still do this? telegram discord itp

**Comment by Inosuke-X1** [comment]
Galera tem algum metodo fucionando 2023? tem gente vendendo contas steam antiga, entao deve ter um novo metodo

**Comment by spalaxez** [comment]
> не за что благодарить

ты русский?

**Comment by SantiARG543** [comment]
i managed to get one today 24/12/2024
